vue-frontend/src/components/partials/HeaderV2.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div class="nav-container" :style="{ height: navContainerHeight + 'px' }">
+    <nav
+      class="navbar navbar-expand-md navbar-light main-header"
+      :class="{
+        'navbar-sticky': navbarSticky,
+        'navbar-animation': navbarAnimation,
+      }"
+      ref="mainHeader"
+    >
+      <div class="container">
+        <a href="/" replace>
+          <div class="navbar-brand">
+            <img
+              :src="headerLogoImage"
+              class="header-logo-image mt-1"
+              alt="canopas-logo"
+            />
+          </div>
+        </a>
+        <div class="navbar-collapse">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <div v-for="navbar in navbars" :key="navbar">
+              <li class="my-2 nav-item-margin">
+                <a
+                  class="v2-normal-3-text"
+                  :class="[
+                    navbar.className,
+                    showContactBtn && navbar.name == `Let's talk`
+                      ? currentRoutePath == `/contact`
+                        ? 'v2-button v2-button-active'
+                        : 'v2-button'
+                      : 'nav-link is-animation-tab',
+                    currentRoutePath == navbar.url
+                      ? showContactBtn && navbar.name == `Let's talk`
+                        ? ''
+                        : 'active-tab'
+                      : '',
+                  ]"
+                  :href="navbar.url"
+                  :target="navbar.target ? '_blank' : ''"
+                  >{{ navbar.name }}</a
+                >
+              </li>
+            </div>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </div>
+</template>
+
+<script type="module">
+import headerLogoImage from "@/assets/images/logo/logo-header.svg";
+import Config from "@/config.js";
+
+export default {
+  data() {
+    return {
+      headerLogoImage: headerLogoImage,
+      navbarSticky: false,
+      navbarAnimation: false,
+      navContainerHeight: 133,
+      lastScrollY: 0,
+      currentRoutePath: this.$router.currentRoute._value.path,
+      showContactBtn: false,
+      navbars: [
+        {
+          name: "Home",
+          url: "/",
+          className: "nav-link is-animation-tab",
+          target: false,
+        },
+        {
+          name: "Career",
+          url: "/jobs",
+          className: "nav-link is-animation-tab",
+          target: false,
+        },
+        {
+          name: "Blog",
+          url: Config.BLOG_URL,
+          className: "nav-link is-animation-tab",
+          target: true,
+        },
+        {
+          name: "Portfolio",
+          url: "",
+          className: "nav-link is-animation-tab",
+          target: false,
+        },
+        {
+          name: "Let's talk",
+          url: "/contact",
+          className: "me-0",
```

Comment:
Remove `me-0 `class if not used

Reply:
not that is need because we no need to margin left after that let's talk button and link and also break that button css also @cp-sumi-k